pysr/sr.py

```diff
@@ -1989,11 +2004,70 @@ def get_hof(self):
             return ret_outputs
         return ret_outputs[0]
 
+    def plot(self, ax, threshold=0.05, fontsize=10):
```

Comment:
I think the function name should be more specific. There are many different types of plots you could want to make. This one is a specific view of the equations.

Reply:
> Thanks, added some comments. Also, after those - could you add a unit-test to make sure it works as expected?

Will do!